doc/tutorials/enable_ptm.rst

```diff
@@ -0,0 +1,67 @@
+.. _enable-ptm:
+
+Enable PCIe Precision Time Management
+#####################################
+
+The PCI Express (PCIe) specification defines a Precision Time Measurement (PTM)
+mechanism that lets you coordinate and synchronize events across multiple PCI
+components within the same system with very fine time precision.
+
+ACRN adds PCIe root port emulation in the hypervisor to support the PTM feature
+and emulates a simple PTM hierarchy.  ACRN enables PTM in a Guest VM if the user
+sets the ``enable_ptm`` option when passing through a device to a post-launched
+VM. When the user enables PTM, the passthru device is connected to a virtual
+root port instead of the host bridge as it normally would.
+
+Here is an example launch script that enables PTM on a passthru Ethernet card:
+
+.. code-block:: bash
+   :emphasize-lines: 5
+
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge \
```

Comment:
Let's add the section showing the `pci-stub`, `bind` and `unbind` operations as well if these are required.

Reply:
@dbkinder Please refer to: 
'''
declare -A passthru_vpid
declare -A passthru_bdf
passthru_vpid=(
["ethptm"]="8086 15f2"
)
passthru_bdf=(
["ethptm"]="0000:aa:00.0"
)
echo ${passthru_vpid["ethptm"]} > /sys/bus/pci/drivers/pci-stub/new_id
echo ${passthru_bdf["ethptm"]} > /sys/bus/pci/devices/${passthru_bdf["ethptm"]}/driver/unbind
echo ${passthru_bdf["ethptm"]} > /sys/bus/pci/drivers/pci-stub/bind
'''